src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,101 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+import { Todo } from '../../types/Todo';
+import { useState, useEffect } from 'react';
+import { Option } from '../Option';
+import { getTodos } from '../../api';
+
+interface Props {
+  setTodos: (value: Todo[]) => void;
+}
+
+type FilterType = 'all' | 'active' | 'completed';
+
+export const TodoFilter: React.FC<Props> = ({ setTodos }) => {
+  const [inputSearch, setInputSearch] = useState<string>('');
+  const [close, setClose] = useState<boolean>(false);
```

Comment:
```suggestion
  const [close, setClose] = useState(false);
```
it's redundant to specify types for primitives